### DISCUSSION TITLE
Simple step by step to build a node 18.15.0 executable

### DISCUSSION THREAD

**Discussion by bycym** [yao-pkg/pkg#54] [General]
Hello there,

Could somebody help me out? I couldn't find any solution on how I could build a version that was released in the pkg-fetch's [release page](https://github.com/vercel/pkg-fetch/releases).

So I tried the `18.15.0` and the `19.8.1` as well, but for example I'll show my steps for node 19.
What I tried:

My package.json's pkg part:
```json
{
    ...
    "bin": "dist/entry.js",
    "pkg": {
        "scripts": "./dist/**/*.js",
        "assets": [
            "./node_modules/**/*",
            "./dist/**/*"
        ],
        "targets": [
            "node19-win-x64",
            "node19-macos-x64",
            "node19-linux-x64"
        ]
    }
}

```

1. get the executable with pkg-fetch
    ```shell
    PKG_IGNORE_TAG=1 node_modules/.bin/pkg-fetch --node-range node19 --output dist/ 
    ```
1. then use the pre-builded executable, that's what I don't get to work:
    ```shell
    PKG_IGNORE_TAG=1 PKG_CACHE_PATH=./dist node_modules/.bin/pkg --output release --public dist/entry.js -t node19.8.1-linux-x64
    ```
What am I doing wrong?

Tons of thanks for your help. :bow:

**Comment by robertsLando** [comment]
only lts versions are supported, use 18 or 20

**Reply by bycym** [reply]
Thank you for the quick answer, how can I do it for 18.15.0 or 20?

**Reply by robertsLando** [reply]
Just put `18/20` where you places `19` in targets

**Reply by bycym** [reply]
I cannot be enough thankful for your package. I forgot to update your package instead of the old vercel. It works as it should. Have a great day my hero!
